pkg/morph/client/notary.go

```diff
@@ -879,10 +879,22 @@ func (c *Client) CalculateNonceAndVUB(hash util.Uint256) (nonce uint32, vub uint
 	// TODO: #1151 cache values since some operations uses same TX as triggers
```

Comment:
delete this line and link to https://github.com/nspcc-dev/neofs-node/issues/1151? or this PR is not about it?